lib/elixir/lib/kernel/typespec.ex

```diff
@@ -864,6 +864,19 @@ defmodule Kernel.Typespec do
     {:op, line(meta), op, {:integer, line(meta), integer}}
   end
 
+  # Handle remote calls in the form of @module_attribute.type.
```

Comment:
I am not 100% sure yet but I lean on thinking backticks in comments are not a good thing. Also, I find this readable tbh, if anyone else finds this a problem I can update. Thanks @eksperimental.